llvm/lib/Transforms/Instrumentation/RealtimeSanitizer.cpp

```diff
@@ -17,47 +17,78 @@
 #include "llvm/IR/IRBuilder.h"
 #include "llvm/IR/Module.h"
 
+#include "llvm/Demangle/Demangle.h"
 #include "llvm/Transforms/Instrumentation/RealtimeSanitizer.h"
 
+#include <vector>
+
 using namespace llvm;
 
+static std::vector<Type *> getArgTypes(ArrayRef<Value *> FunctionArgs) {
+  std::vector<Type *> Types;
```

Comment:
Prefer SmallVector instead of std::vector, even if you don't want to use inline storage.

Reply:
Thank you!